samples/net/sockets/sntp_client/sample.yaml

```diff
@@ -3,9 +3,8 @@ sample:
   name: sntp_client
 common:
   harness: net
-  platform_allow: qemu_x86
+  platform_allow: qemu_x86 native_posix
   tags: net
-  filter: TOOLCHAIN_HAS_NEWLIB == 1
 tests:
```

Comment:
This should actually now build w/o any POSIX support being enabled at all, so maybe add a "raw" configuration as well?